hack/testing-olm/test-657-im-block-autocreate-for-write-suffix.sh

```diff
@@ -14,7 +14,7 @@ if [ ! -d $test_artifact_dir ] ; then
   mkdir -p $test_artifact_dir
 fi
 
-TEST_NAMESPACE="${TEST_NAMESPACE:-e2e-test-${RANDOM}}"
+TEST_NAMESPACE="openshift-operators-redhat"
```

Comment:
we should be hardcoding here instead: https://github.com/openshift/elasticsearch-operator/pull/317/files#diff-060831063a2c1534f1514647016229a2R48

Reply:
> we should be hardcoding here instead: https://github.com/openshift/elasticsearch-operator/pull/317/files#diff-060831063a2c1534f1514647016229a2R48

We can revert this change as the latest test exhibits the same failure regardless of NS.  I presume since we fixed the validation problems it must be attributed to something else